src/filters/humanDate.js

```diff
@@ -5,5 +5,5 @@
  * @return {string}
  */
 export default function (ts) {
-  return new Date(ts).toISOString()
+  return new Date(ts).toString().split(' ').slice(1, 5).join(' ')
```

Comment:
```suggestion
  return new Date(ts).toLocaleString()
```
is it fits in this case? Or even ```.toLocaleString('en-US', { month: 'short', day: 'numeric', year: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false })```. I think to use `toLocaleString` is more obvious than spliting and slicing.